analyses/mutational-signatures/03-de_novo_range_of_nsignatures.sh

```diff
@@ -36,23 +49,40 @@ then
     --num_iterations 10 \
     --seed 42 
 else
-  for model in multinomial poisson; do
-    for seed in {1..5}; do
+  # GOF
+  if [[ $ANALYSIS -eq "0" ]] 
```

Comment:
Once we get further down the line, I imagine that the module documentation will do a lot of heavy-lifting around these two steps (GOF, analysis).

Reply:
Yes, definitely. I thought this was the cleanest way to do the two steps that refer to the same R script. Note, this R script would NOT be used if we also proceed to fit the 8 CNS tumors.